module/src/components/tabSelect/tabControl/tabControl.animated.scss

```diff
@@ -0,0 +1,4 @@
+.arm-tab-control-tab {
+  transition-duration: $transition-quick;
+  transition-property: background-color, color, border-color;
+}
```

Comment:
This comment is for the whole `components/tabControl` directory. 

Should it not be removed altogether according to the tribunal spreadsheet or have I misunderstood?

(I'll scan the code for changes anyway in case it stays)

Reply:
So, components/tabControl should come out, but components/tabSelect/tabControl is a trickier one. We could refactor tabControl to exist purely inside tabSelect as it is used there, this was the expedient version of that fix. Either refactor here or, I guess, raise a ticket to do so?

Reply:
Agreed to leave this as is and pick it off in one of the later clean-up tickets as needed.